webpack.config.js

```diff
@@ -16,150 +14,147 @@ const mediaQueryAliases = {
+      'country-selector': './assets/src/scss/partials/country-selector.scss',
+      'gravity-forms': './assets/src/scss/layout/_gravity-forms.scss',
+      'gravityforms-client-side': './assets/src/js/gravityforms-client-side.js',
+      media_archive: './assets/src/js/media_archive.js',
+      media_archive_editor_view: './assets/src/js/media_archive_editor_view.js',
+      'lite-yt-embed': './node_modules/lite-youtube-embed/src/lite-yt-embed.js',
+      menu_editor: './assets/src/js/menu_editor.js',
+      frontendIndex: './assets/src/frontendIndex.js',
+      editorIndex: './assets/src/editorIndex.js',
+      ArticlesScript: './assets/src/blocks/Articles/ArticlesScript.js',
+      ArticlesEditorScript: './assets/src/blocks/Articles/ArticlesEditorScript.js',
+      GalleryScript: './assets/src/blocks/Gallery/GalleryScript.js',
+      GalleryEditorScript: './assets/src/blocks/Gallery/GalleryEditorScript.js',
+      GalleryStyle: './assets/src/scss/blocks/Gallery/GalleryStyle.scss',
+      GalleryEditorStyle: './assets/src/scss/blocks/Gallery/GalleryEditorStyle.scss',
+      GuestBookScript: './assets/src/blocks/GuestBook/GuestBookScript.js',
+      GuestBookEditorScript: './assets/src/blocks/GuestBook/GuestBookEditorScript.js',
+      CarouselHeaderScript: './assets/src/blocks/CarouselHeader/CarouselHeaderScript.js',
+      CarouselHeaderEditorScript: './assets/src/blocks/CarouselHeader/CarouselHeaderEditorScript.js',
+      AccordionScript: './assets/src/blocks/Accordion/AccordionScript.js',
+      AccordionEditorScript: './assets/src/blocks/Accordion/AccordionEditorScript.js',
+      CookiesScript: './assets/src/blocks/Cookies/CookiesScript.js',
+      CookiesEditorScript: './assets/src/blocks/Cookies/CookiesEditorScript.js',
+      CounterScript: './assets/src/blocks/Counter/CounterScript.js',
+      CounterEditorScript: './assets/src/blocks/Counter/CounterEditorScript.js',
+      CoversEditorScript: './assets/src/blocks/Covers/CoversEditorScript.js',
+      CoversScript: './assets/src/blocks/Covers/CoversScript.js',
+      SpreadsheetScript: './assets/src/blocks/Spreadsheet/SpreadsheetScript.js',
+      SpreadsheetEditorScript: './assets/src/blocks/Spreadsheet/SpreadsheetEditorScript.js',
+      SocialMediaEditorScript: './assets/src/blocks/SocialMedia/SocialMediaEditorScript.js',
+      SocialMediaStyle: './assets/src/scss/blocks/SocialMedia/SocialMediaStyle.scss',
+      SocialMediaEditorStyle: './assets/src/scss/blocks/SocialMedia/SocialMediaEditorStyle.scss',
+      TimelineScript: './assets/src/blocks/Timeline/TimelineScript.js',
+      TimelineEditorScript: './assets/src/blocks/Timeline/TimelineEditorScript.js',
+      TimelineStyle: './assets/src/scss/blocks/Timeline/TimelineStyle.scss',
+      TimelineEditorStyle: './assets/src/scss/blocks/Timeline/TimelineEditorStyle.scss',
+    },
+    output: {
+      filename: '[name].js',
+      path: __dirname + '/assets/build',
+    },
+    module: {
```

Comment:
Is `...defaultConfig.module` no longer needed?

Reply:
Yes!